### PR TITLE
uftrace: Stringify default values for some options

### PR DESCRIPTION
**Pull request by honggyukim**
It's better to avoid hard coding for default values for some options.

Signed-off-by: Honggyu Kim <honggyu.kp@gmail.com>

### CODE REVIEW

File: libmcount/mcount.h [hunk 1]
```diff
@@ -66,7 +66,8 @@ void _mcleanup(void);
 void mcount_restore(void);
 void mcount_reset(void);
 
-#define SHMEM_BUFFER_SIZE  (128 * 1024)
+#define SHMEM_BUFFER_KB_SIZE	128
+#define SHMEM_BUFFER_SIZE	(SHMEM_BUFFER_KB_SIZE * KB)
```
Comment (namhyung) [inline libmcount/mcount.h]: Just nitpick, I prefer `SHMEM_BUFFER_SIZE_KB`

File: uftrace.h [hunk 2]
```diff
@@ -30,6 +30,9 @@
 #define KB 1024
 #define MB (KB * 1024)
 
+#define stringify(s)    __stringify(s)
+#define __stringify(s)  #s
+
```
Comment (namhyung) [inline uftrace.h]: Could you please move them to "utils.h"?

File: libmcount/mcount.h (hunk 1 above)
Comment (honggyukim) [reply libmcount/mcount.h]: done.

File: uftrace.h (hunk 2 above)
Comment (honggyukim) [reply uftrace.h]: done.

**Comment by honggyukim** [comment]
@namhyung Thanks for the review. I applied your comments and also added one more commit for `-s`/`--sort` value.

**Comment by honggyukim** [comment]
Please wait. I will just polish some commits and update again.

**Comment by honggyukim** [comment]
@namhyung Please take a look at it when you have time. Thanks.

**Pull request merged**